cypress/e2e/dashboard/events/events-index.cy.ts

```diff
@@ -297,23 +297,50 @@ describe('events dashboard', () => {
 
   it('chapter admin should be allowed to edit event, but nobody else', () => {
     const eventId = 1;
+
+    cy.logout();
+    // newly registered user (without a chapter_users record)
+    cy.register();
+
+    cy.login(Cypress.env('JWT_TEST_USER'));
+    cy.reload();
+    cy.updateEvent(eventId, eventData).then(expectToBeRejected);
+    cy.logout();
+    // banned admin should be rejected
+    cy.login(Cypress.env('JWT_BANNED_ADMIN_USER'));
+    cy.reload();
+    cy.updateEvent(eventId, eventData).then(expectToBeRejected);
+
+    cy.logout();
     // admin of chapter 1
     cy.login(Cypress.env('JWT_ADMIN_USER'));
     cy.reload();
     cy.updateEvent(eventId, eventData).then((response) => {
       expect(response.body.errors).not.to.exist;
     });
+  });
+
+  it('chapter admin should be allowed to delete event, but nobody else', () => {
+    const eventId = 1;
+
     cy.logout();
     // newly registered user (without a chapter_users record)
     cy.register();
 
     cy.login(Cypress.env('JWT_TEST_USER'));
     cy.reload();
-    cy.updateEvent(eventId, eventData).then(expectToBeRejected);
-    cy.logout();
+    cy.deleteEvent(eventId).then(expectToBeRejected);
+
     // banned admin should be rejected
     cy.login(Cypress.env('JWT_BANNED_ADMIN_USER'));
     cy.reload();
-    cy.updateEvent(eventId, eventData).then(expectToBeRejected);
+    cy.deleteEvent(eventId).then(expectToBeRejected);
+    cy.logout();
+    // admin of chapter 1
+    cy.login(Cypress.env('JWT_ADMIN_USER'));
```

Comment:
```suggestion
    cy.login(Cypress.env('JWT_CHAPTER_1_ADMIN_USER'));
```
This *does* work as is, but only because it's effectively `cy.login(undefined)` which defaults to the instance owner.  To be honest, I think that default should go and we should be explicit about who we're logging in as each time. Not that we need to fix that here!